ansible/vagrant/Vagrantfile

```diff
@@ -6,6 +6,7 @@ require "yaml"
 ### RUN: vagrant plugin uninstall vagrant-openstack-plugin"
 ### Then RUN: "vagrant plugin install vagrant-openstack-provider"
 require 'vagrant-openstack-provider'
+require 'vagrant-hostmanager'
```

Comment:
I was using this for vbox, updates etc/hosts dynamically.    not sure if it needs to be in req unless it works for other VMs to ( i think it could be useful for os-provisioner also?)